kernel/sched.c

```diff
@@ -414,30 +414,45 @@ static void flag_ipi(void)
 
 static int slice_ticks;
 static int slice_max_prio;
-struct _timeout slice_timeouts[CONFIG_MP_MAX_NUM_CPUS];
-bool slice_expired[CONFIG_MP_MAX_NUM_CPUS];
+static struct _timeout slice_timeouts[CONFIG_MP_MAX_NUM_CPUS];
+static bool slice_expired[CONFIG_MP_MAX_NUM_CPUS];
```

Comment:
Oops.  I do still think that these naturally ought to be part of the CPU struct.